src/ui/breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,28 @@
+import { Link, useLocation } from 'react-router-dom';
+import { Screens } from '@/ui/App';
+import { useSessionContext } from '@/ui/session/SessionProvider';
+
+export function Breadcrumbs( props: { className: string } ) {
+	const { className } = props;
+	const { session } = useSessionContext();
+	const location = useLocation();
+
+	const showSession = location.pathname.startsWith( '/session' );
+
+	return (
+		<nav className={ className }>
+			<ul>
+				<li>
+					<Link to={ Screens.home() }>Home</Link>
+				</li>
+				{ ! showSession ? null : (
+					<li>
+						<Link to={ Screens.viewSession( session.id ) }>
+							{ session.title }
```

Comment:
```suggestion
							Session: { session.title }
```

Reply:
How about we indicate it's a session for clarity?

Reply:
I see what you mean, maybe it could be `Home -> Sessions -> Alex Kirk`? But at the moment there is no sessions screen because the list of sessions gets rendered directly in the home.

A couple of other things to bear in mind is that the UI real estate here is limited (specially given that the site title can be long, we will probably need to truncate it in some way), and also that at the moment we aren't exposing the term "session" to the user yet, which I think is maybe a good thing.

If that's ok with you I would leave it as is for now and we can re-evaluate next time we expand the breadcrumbs, which I think we will need to do soon (e.g. `Home -> Alex Kirk -> Import Blog Posts`)

Reply:
I am ok with leaving as is for now and revisiting afterwards 👍